Create 2021-09-24-september-newsletter.md

anisa-hawes opened the pull request:
I am creating our September Newsletter.

Closes #2221 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

anisa-hawes committed:
Create 2021-09-24-september-newsletter.md

I am creating our September Newsletter.

jenniferisasi commented:
ah, that's an easy fix. Each author has to go on its own line with the hyphen. 

I am going to read the text and all to see if there are any changes I want to do before initiating the build.

anisa-hawes commented:
Ah! Thank you, Jenn! I didn't remember that! I'll make the change.

anisa-hawes committed:
Update 2021-09-24-september-newsletter.md

Adjusting YAML header so that authors are listed in the correct format:

`authors:
- Forename Surname
- Forename Surname`

anisa-hawes committed:
Update ph_authors.yml

I am adding a bio (EN only for now)

anisa-hawes committed:
Update ph_authors.yml

Correcting typing error.

jenniferisasi committed:
Update 2021-09-24-september-newsletter.md
